### PR TITLE
Views

### PR DESCRIPTION
**Pull request by diegoh40**
- Use preprocessed HTML file with embedded Ruby code.
- Use layouts and templates for shared content.

### CODE REVIEW

File: app/views/users/show.html.erb [hunk 1]
```diff
@@ -1 +1,26 @@
...
+<p>Numero of  posts:<%= @user.posts_counter %></p>
+</div>
+</div>
+ <div class="bio">  
+ <h4>Bio</h4>
+   <h1> <%= @user.bio %> </h1>
+</div>
+
+
+<% @posts.each_with_index do |post, index|%>
```
Comment (FrederickMih) [inline app/views/users/show.html.erb:15]: - [ ] Based on the requirements of this project, we are supposed to retrieve and render the user's three most recent posts. Kindly fix :wrench: this. 
- **Hint** `<% @user.last_three.each do |post, index|%>`

- [ ] As mentioned above, you need a link(`See all Post`) at the bottom of the page. Clicking on the link should display all the user's posts as seen on this [wireframe](https://github.com/microverseinc/curriculum-rails/raw/main/blog-app/images/blog_user_all_posts.png)

File: app/views/posts/show.html.erb [hunk 2]
```diff
@@ -1 +1,22 @@
...
+  <div class="cont-comments">
+
+    <% @post.comments.each do |comment|%>
+      <p><%= comment.author.name %>: <%= comment.text %></p>
+    <% end %>
+    
+  </div>
+  </main>
+
+
```
Comment (FrederickMih) [inline app/views/posts/show.html.erb:22]: - Once more, let's make our views look presentable as seen on the wireframes.

**Review by FrederickMih — CHANGES_REQUESTED** [review]
## Hi :raising_hand: @diegoh40 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!




## Highlights


- PR is descriptive :+1: 
- The five most recent comments were display on the post index page :+1: 
- No Linter error :heavy_check_mark: 


## Required Changes ♻️

- Let's kindly get rid of the Microverse logo on our README file. You can add your own logo or leave it empty. Also, fill the **? FAQ** section to make your README looks more descriptive and professional

_Check the comments under the review._

## Optional suggestions

- [**Optional**] To avoid the Rails default page from showing up when points to the URL(http://localhost:3000/), let's set the root routes to the index view page(`root "users#index"`) in the `routes.rb` file.

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: app/views/users/index.html.erb [hunk 3]
```diff
@@ -1 +1,14 @@
...
+<% @users.each do |user|%>
+<div class="container">
+<img src='<%= user.photo %>' alt='<%= user.name %> thumbnail' />
+ <%= link_to(user) do %>
+<div class="info">
+   <h1> <%= user.name %> </h1>
+<p>Number of posts:<%= user.posts_counter %></p>
+</div>
+<% end %>
+</div>
```
[the remark this answers is not in the document]
Comment (diegoh40) [reply app/views/users/index.html.erb:11]: Hi thanks for the advice but, if you notice the variable is different on each page, I should't do the same for all of them.

**Review by cynthiainga — CHANGES_REQUESTED** [review]
Hi @diegoh40, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't been addressed yet.



 ## Highlights

- [ ] No linters errors ✔️ 
- [ ] Descriptive PR ✔️ 
- [ ] Correct Gitflow ✔️ 

 ## Required Changes ♻️

- [ ] Please kindly check the comments under the review 👇 👇 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by sumairq — APPROVED** [review]
### Hi @diegoh40  , 



Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
✔️ Good commit history
✔️Linters are passing
✔️Very professional README
✔️Correct Git flow
✔️Descriptive PR
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**